plugins/shadow-block-converter/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+      "baseUrl": "./",
+      "outDir": "dist",
+      "declaration": true,
+      "declarationMap": true,
+      "downlevelIteration": true,
```

Comment:
Remove `downlevelIteration`: It's only useful if you expect this code to be used in an ES5 environment where a `Symbol.iterator` implementation *might( exist. Since we're compiling to ES2015, it's not necessary.

Reference: https://www.typescriptlang.org/tsconfig#downlevelIteration

Reply:
Done. This file was copied verbatim from the field-slider example. I've taken the liberty of applying the same change to the example I took this from.